scripts/integration-tests/utils/e2e-check.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-process-exit */
+import fs from "fs";
+import fetch from "node-fetch";
+
+const resp = await fetch(
+  "https://raw.githubusercontent.com/babel/babel/main/package.json"
+);
+const mainRepoPackageJson = await resp.json();
+
+if (
+  mainRepoPackageJson.version !==
+  JSON.parse(fs.readFileSync("package.json")).version
+) {
+  if (process.env.GITHUB_REPOSITORY === "babel/babel") {
+    console.error("The branch is not up to date with main, please rebase");
+    process.exit(1);
```

Comment:
```suggestion
    process.exitCode = 1
```
Because printing to `stdout` could be asynchronous: https://nodejs.org/api/process.html#processexitcode

Reply:
This is also prevented by the rule https://eslint.org/docs/latest/rules/no-process-exit, I was upset that this rule is disabled in this file for no apparent reasons.